dgfit/obsdata.py

```diff
@@ -42,18 +42,21 @@ class ObsData(object):
     """
 
     # read in the data from files
-    def __init__(
-        self,
-        obs_filename,
-    ):
+    def __init__(self, obs_filename, path="./"):
+
+        if path != "./":
+            obs_filename = path + obs_filename
```

Comment:
The if statement should be removed.  The code on line 48 will work correctly even in the default case.